.github/ISSUE_TEMPLATE/release-tracking-issue.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Release Tracking Issue
+about: Handy template for release tracking
+title: 'Releases: '
+labels: release
+assignees: ''
+
+---
+
+## KEY
+**✏️ –– PR opened**
+
+---
+## {RELEASE_VERSION_NUMBER}
+- [ ]  Check for new project versions
+  - [ ] kubernetes/kubernetes
+  - [ ] containernetworking/plugins
+  - [ ] coredns/coredns
+  - [ ] kubernetes-sigs/metrics-server
+  - [ ] kubernetes-sigs/aws-iam-authenticator
+  - [ ] etcd-io/etcd
+  - [ ] kubernetes-csi/external-snapshotter
+  - [ ] kubernetes-csi/external-attacher
+  - [ ] kubernetes-csi/external-provisioner
+  - [ ] kubernetes-csi/external-resizer
+  - [ ] kubernetes-csi/livenessprobe
+  - [ ] kubernetes-csi/node-driver-registrar
+- [ ] Check for patch changes
+- [ ] Merge bot's attribution PR
```

Comment:
```suggestion
```

Reply:
This is something we should just do and isn't specific to the release per se. I called it out last time because Jackson wanted this before the release, and we were waiting on it.